commit/report.go

```diff
@@ -323,13 +517,13 @@ func (p *Plugin) ShouldTransmitAcceptedReport(
 ) (bool, error) {
 	ctx, lggr := logutil.WithOCRInfo(ctx, p.lggr, seqNr, logutil.PhaseShouldTransmit)
 
-	valid, decodedReport, err := p.validateReport(ctx, lggr, seqNr, r)
-	if err != nil {
+	decodedReport, err := p.validateReport(ctx, lggr, seqNr, r)
+	if errors.Is(err, plugincommon.ErrValidationError) {
```

Comment:
Are these new changes covered by tests?

Reply:
Yes, there are many tests for the report validation function. Many of them failed until I got this logic just right.